templates/docker/env/local.env.dist

```diff
@@ -30,12 +30,12 @@ MAIL_PORT=1025
 
 ## Xdebug config
 XDEBUG_IDE_KEY=PHPSTORM
-XDEBUG_CONFIG=remote_host=10.254.254.254
+XDEBUG_CONFIG=remote_host=docker.for.mac.host.internal
```

Comment:
@shakyShane Is this the same as `docker.for.mac.localhost`

Reply:
works the same for now, but Docker advise against `docker.for.mac.localhost` now,

`docker.for.mac.host.internal` ftw

Reply:
👍